src/mem/phymem.c

```diff
@@ -44,10 +39,28 @@ static int phymem_init(void) {
 	return phymem_alloc_start == va ? 0 : -EIO;
 }
 
+char * const phymem_allocated_start(void) {
+	extern char _reserve_end;
+
+    return (char *) binalign_bound((uintptr_t) &_reserve_end, PAGE_SIZE());
```

Comment:
Something wrong with tabs/spaces in this function.

Reply:
Fixed.